### ISSUE TITLE
go2rtc not working on any version above 1.2.0

### ISSUE THREAD

**Issue by paulooliveira** [AlexxIT/go2rtc#905]
Hi Alexx,

First of all, tnx for the great work!
I'm trying to upgrade from version 1.2.0 to a version >= 1.5.0 to try and get 2-way audio on the TP-Link Vigi cameras.
Currently on this lab setup I have only one C340 connected
With 1.2.0 the webrtc works perfectly, but as soon as I try any version above that, only MSE keeps working (with 1.2.0 both webrtc and MSE work perfectly and with sound)

This file is from when I was testing the 1.3.2 but anything above 1.2.0 has the same results
I attach both versions of the main stream info:
[1.2.0.txt](https://github.com/AlexxIT/go2rtc/files/14067191/1.2.0.txt)
[1.3.2.txt](https://github.com/AlexxIT/go2rtc/files/14067192/1.3.2.txt)

And my go2rtc config:
```
api:
  origin: '*'
log:
  format: text
rtsp:
  default_query: mp4
streams:
  front_camera:
  - rtsp://admin:xxxx&@192.168.1.137:554/stream1
  - ffmpeg:front_camera#audio=aac
  front_camera_sub:
  - rtsp://admin:xxxx&@192.168.1.137:554/stream2
webrtc:
  candidates:
  - stun:8555
```

No errors are being reported anywhere.
The behaviour is the same on Chrome, Firefox or Safari
Basically is the live webrtc that is not working, the Frigate recording continues to work well, but the live webrtc is never displayed, I can only see the recordings.
I know I mentioned Frigate but the behaviour is exactly the same on the go2rtc dashboard

I've run out of ideas to fix it, can you help? Tnx!

**Comment by paulooliveira** [comment]
Ok, I feel a bit dumb now... (just a bit) 🫣

Somehow, all the versions above 1.2.0 really need the IP defined on the candidates
```
webrtc:
  candidates:
  - 192.168.1.40:8555
  - stun:8555
```
I didn't had it on the 1.2.0 config and it worked perfectly, not anymore

Now with the server IP configured everything works correctly, even with 1.8.5
I will close the ticket, this can stay here as a future reference to others 👍
